### PR TITLE
Have a GSoC student post

### PR DESCRIPTION
**Pull request by kain88-de**
I kind of forgot to make a blog post about this earlier.

@utkbansal Could you provide us with a short paragraph for your biography? You can have a look what our last students wrote last year about themselves [here](http://www.mdanalysis.org/2016/04/26/gsoc/)

### CODE REVIEW

File: _posts/2017-06-03-gsoc-students.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Google Summer of Code Students 2017
+---
+
+We are happy to anounce that MDAnalysis is hosting a [GSoC][gsoc] students for
```
Comment (orbeckst) [inline _posts/2017-06-03-gsoc-students.md]: student (singular)

File: _posts/2017-06-03-gsoc-students.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Google Summer of Code Students 2017
```
Comment (orbeckst) [inline _posts/2017-06-03-gsoc-students.md]: student (singular)

File: _posts/2017-06-03-gsoc-students.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+We are happy to anounce that MDAnalysis is hosting a [GSoC][gsoc] students for
+the [NumFOCUS][numfocus] this year, [Utkarsh Bansal][utkarsh-gsoc].
```
Comment (orbeckst) [inline _posts/2017-06-03-gsoc-students.md]: I don't like having the name link to the project description, I'd rather do something like

     ... this year, [Utkarsh Bansal](http://utkarshbansal.me/about/) (@utkbansal on GitHub), with his project ["Port to pytest"][utkarsh-gsoc].

so any link to his name points to something related to him as a person and it is also clear what his project is.

File: _posts/2017-06-03-gsoc-students.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# Utkarsh Bansal: Port unit tests to pytest
+
+<img
+src="https://avatars2.githubusercontent.com/u/8345336?v=3&s=460"
+title="Utkarsh Bansal" alt="Utkarsh Bansal"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Utkarsh will port our complete unit tests from nosetest to pytest. This is a
```
Comment (orbeckst) [inline _posts/2017-06-03-gsoc-students.md]: Maybe add links to [nose](http://nose.readthedocs.io) (not "nosetest") and [pytest](https://pytest.org)

**Comment by kain88-de** [comment]
@utkbansal please provide us a short bio you would like to have published.

**Comment by utkbansal** [comment]
I am currently pursuing my bachelors in Computer Science and Engineering and will be graduating this summer. I hope to learn a lot here and plan to continue my career as a software developer where I can put all my learnings to good use.

**Comment by utkbansal** [comment]
Also, my blog is available at [utkarshbansal.me](utkarshbansal.me)

File: _posts/2017-06-03-gsoc-students.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Google Summer of Code Student 2017
+---
+
+We are happy to anounce that MDAnalysis is hosting a [GSoC][gsoc] student for
+the [NumFOCUS][numfocus] this year, [Utkarsh Bansal][utkarsh-blog] (@utkbansal
```
Comment (orbeckst) [inline _posts/2017-06-03-gsoc-students.md]: ~~the~~

**Pull request merged**